scripts/build_docs.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2019 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+#
+
+# Main Documentation build script
+# Syntax:
+#
+# build_docs --url=TARGET_URL           # url where the docs will live, e..g https://mysite.com
+#            --url-path=TARGET_PATH     # target path on doc site, e.g. /developer_docs
+#            --source=SOURCE_FOLDER     # source location
+#            --output=OUTPUT_FOLDER     # build target. This folder will be deleted by the script.
+
+# exit on error
+set -e
+
+for i in "$@"
```

Comment:
I think my comment about shell script vs python script applies even more here. I have no idea what this for loop does.

Reply:
For this script, i disagree. Python is not the right choice for a script that basically executes a series of commands. This runs inside a docker container, so no need to think about windows either. I can add a comment, but this is very standard bash script.